services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java

```diff
@@ -39,10 +39,10 @@ void testGetContentTypeZip(String path) {
   @ParameterizedTest
   @ValueSource(strings = {
       "version/v1/diagnosis-keys/country/DE/date/2020-06-11",
-      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour/13" })
+      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour/13"})
   void testIsKeyFile(String path) {
     LocalFile test = new LocalIndexFile(Path.of("/root", path, "/index"), Path.of("/root"));
-    assertTrue(test.isKeyFile());
+    Assertions.assertTrue(test.isKeyFile());
```

Comment:
I personally am in favour of the `import static` to use assertions directly. But anyway, this is also a formatting thing and not relevant for your PR. :-)

Reply:
I dont know when I shoul take in consideration what IntelliJ recommends and when not. In this case, I had this worning: Call to 'assertTrue()' from 'Assert' should be replaced with call to method from 'org.junit.jupiter.api.Assertions'